### PR TITLE
Few spec file changes

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

File: dist/obs-server.spec [hunk 1]
```diff
@@ -483,7 +483,7 @@ for i in production.rb ; do
 done
 
 if [ ! -e %{secret_key_file} ]; then
-  ( umask 0077; dd if=/dev/urandom bs=256 count=1 2>/dev/null |sha256sum| cut -d\  -f 1 > %{secret_key_file} )
+  ( umask 0077; RAILS_ENV=production bundle exec rake secret > %{secret_key_file} )
```
Comment (bgeuken) [inline dist/obs-server.spec]: @adrianschroeter Second try;-)

File: dist/obs-server.spec [hunk 2]
```diff
@@ -680,11 +680,13 @@ chown %{apache_user}:%{apache_group} /srv/www/obs/api/log/production.log
 %dir /etc/apache2/vhosts.d
 %config(noreplace) /etc/apache2/vhosts.d/obs.conf
 
+%defattr(0644,wwwrun,root)
```
Comment (adrianschroeter) [inline dist/obs-server.spec]: they should belong to group www as well

File: dist/obs-server.spec (hunk 1 above)
Comment (adrianschroeter) [inline dist/obs-server.spec]: rake might not be using ruby 2.3 by default, so this might break on systems with a default ruby != 2.3

**Comment by adrianschroeter** [comment]
still don't like to call ruby in %post, but hey, let's try it ...

**Pull request merged**